portfolio/src/main/webapp/style.css

```diff
@@ -91,4 +97,44 @@ instead of next to each other */
     flex: 100%;
     max-width: 100%;
   }
+}
+
+@media screen and (max-height: 600px) {
+  #comments-grid {
+    height: 300px
+  }
+} 
+
+@media screen and (max-height: 2000px) {
+  #comments-grid {
+    height: 600px
+  }
+} 
+
+#comments {
+  margin: auto;
+  margin-top: 55px;
```

Comment:
I tend to prefer not mixing `margin` and `margin-...`. Doing `margin: 55px auto auto;` here is probably the cleanest.